src/warnet/warnet.py

```diff
@@ -159,27 +159,21 @@ def tanks_from_graph(self):
                 raise Exception(
                     f"Node ID in graph must be incrementing integers (got '{node_id}', expected '{len(self.tanks)}')"
                 )
-            self.tanks.append(Tank.from_graph_node(node_id, self))
+            tank = Tank.from_graph_node(node_id, self)
+            # import edges as list of destinations to connect to
+            for edge in self.graph.edges(data=True):
```

Comment:
Seems like it should be possible to do something like:

```python
for edge in self.graph.nodes[i].edges:
```

Reply:
In fact, this is changed again in the subsequent commit. Perhaps just squash?

Reply:
added in https://github.com/bitcoin-dev-project/warnet/pull/285/commits/d51d0832662d2965c235401c209c56daa7c14719